SrsLia.py

```diff
@@ -60,7 +61,7 @@
 class SrsLia:
 	def __init__(self,address):
 		self.Address = address
-		self.Visa = VisaSubs.InitializeGPIB(address,0)
+		self.Visa = VisaSubs.InitializeGPIB(address,0,term_chars = "\\n")
```

Comment:
Need to add '\\n" add the end of all serial comment seems to be unique to Janis. Need note this difference.